packages/core/utils/src/index.ts

```diff
@@ -298,3 +304,23 @@ export function isConstructor(obj) {
 export function isEs6Module(module) {
   return (typeof module === 'function' && isConstructor(module)) || (typeof module === 'object' && typeof module.default === 'function' && module.__esModule);
 }
+
+export function warnIfPackageNotDefinedLocally(packageName, warnFunc) {
+  const packageIsResolvable = findUp.sync("node_modules/" + packageName, {cwd: dappPath()});
+  if (!packageIsResolvable) {
+    return warnFunc("== WARNING: it seems "  + packageName + " is not defined in your dapp package.json; In future versions of embark this package should be a local dependency and configured as a plugin");
```

Comment:
Does this warning message need to be revised? The package might or might not be specified in `package.json` but the problem was that it couldn't be resolved relative to dapp's path. An example of when it would be specified in `package.json` but not resolvable is when the user adds it to `package.json` but then forgets to run `npm|yarn install` before running another embark command.